### PR TITLE
ft- designed and savings target card

### PR DESCRIPTION
**Pull request by Dallat65**
**Description of PR**

Design savings target card
Screen Shot 2023-12-25 at 13 32 11
Screen Shot 2023-12-25 at 13 32 00

---

Please check all that apply.

#### Checklist:

- [ ] Rebased with master
- [ ] PR Description provided
- [ ] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [ ] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [ ] Does the endpoint return the correct success/error format?

---

**Screenshots**  
_add relevant screenshots here_

---

**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

**Comment by KristofaJosh** [comment]
Task linked: [CU-86bwmcqe7 Savings target snippet](https://app.clickup.com/t/86bwmcqe7)

**Review by Elyzsium — COMMENTED** [review]
you are not complete your PR description

File: apps/client/src/components/compounds/SavingsTargetCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import styles from './savingsTarget.module.scss'
+import { Text } from 'components'
+
+function formatNaira(amount: number): string {
+  const formattedAmount = amount.toLocaleString('en-NG', {
+    style: 'currency',
+    currency: 'NGN',
+  })
+  return formattedAmount.replace('NGN', '₦')
+}
```
Comment (KristofaJosh) [inline apps/client/src/components/compounds/SavingsTargetCard/index.tsx:10]: i think this exist in the codebase already somewhere...
if yes, move this to the utility function and import it here
also do same for where it was used within the code

File: apps/client/src/components/compounds/SavingsTargetCard/Images/Travel_Img.svg
Comment (KristofaJosh) [inline apps/client/src/components/compounds/SavingsTargetCard/Images/Travel_Img.svg:1]: you can delete this and use
https://picsum.photos/200/300 in the src

**Review by KristofaJosh — APPROVED** [review]

**Review by Elyzsium — APPROVED** [review]

**Review by Elyzsium — APPROVED** [review]

File: apps/client/src/pages/Dashboard/Savings.tsx [hunk 2]
```diff
@@ -1,5 +1,20 @@
+import SavingsTargetCard from 'components/compounds/SavingsTargetCard'
+
 const SavingsPage = () => {
-  return <div>Savings Page</div>
+  return (
+    <div>
+      Savings Page
```
Comment (KristofaJosh) [inline apps/client/src/pages/Dashboard/Savings.tsx:6]: I missed this. 

This isn't in any HTML element/tag 

a P or an H maybe?

**Review by KristofaJosh — APPROVED** [review]

**Pull request merged**